index.js

```diff
@@ -38,6 +113,14 @@ module.exports = {
         data = fs.readFileSync(input.data).toString();
         return validate(data);
       }
+      else if (input.type === 'folder') {
+        if (!_.isEmpty(parse.getRootFiles(input.data))) {
+          return {
+            result: true,
+            reason: 'valid input type'
+          };
+        }
```

Comment:
else?

Reply:
Then the last return will execute with `{result: false, reason: 'input type is not valid'}`